x-pack/legacy/plugins/siem/public/apps/start_app.tsx

```diff
@@ -4,6 +4,8 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+/* eslint-disable react/display-name */
```

Comment:
May want to add a comment here as to why this is being disabled -- when I added `displayName`'s for the below three components (`AppPluginRoot`, `StartApp`, and `SiemApp`) they were not being shown in the react dev tools Component tree, so I assume this is the reason?